content/blog/2023/08/camel4-whatsnew/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Apache Camel 4 What's New (top 10)"
+date: 2023-08-15
+authors: [davsclaus,gzurowski,opiske]
+categories: ["Releases"]
+preview: Top 10 of what's new in Apache Camel version 4
+---
+
+After 10 months of development, with 3 milestone, and 2 RC releases, then Apache Camel v4 is released
+today as LTS release. The Camel `4.0.x` is a LTS release that is supported for 1 year.
+
+This blog post highlights some noteworthy new features and improvements in Camel v4.
+
+The features are based on work since January 2023, which was the time, where we switched `main` branch
+to be Camel v4 based, and our effort was focused primary on Camel v4.
+
+### 1) Major Goals
+
+The need for Camel 4 is mainly driven by Java open source projects migrating from `javax` to `jakarta` APIs
+and to keep up with popular runtimes such as Spring Boot and Quarkus.
+
+1. Migrate from `javax` -> `jakarta` (JEE 10)
+2. Java 17 as minimum
+3. Spring Boot 3
+4. Quarkus 3
+
+Camel 4 requires Java 17. Support for Java 21 is planned for next LTS released by end of this year. 
+
+### 2) Dependency updates
+
+We have upgraded all 3rd party dependencies to their latest releases where possible.
+
+### 3) Performance optimizations
+
+We have improved the performance of the internal Camel routing engine and framework. 
+
+You can find more details in this blog post [camel 4 performance improvements](/blog/2023/05/camel-4-performance-improvements.html).
+
+### 4) Camel Spring Boot
+
+Camel 4 is at the time of release, tested with the latest Spring Boot 3.1.2 release.
+
+This release also comes with our first basic (and very limited) support for Spring Boot native (AOT compilation).
+See more in the [aot-basic](https://github.com/apache/camel-spring-boot-examples/tree/main/aot-basic) example.
+
+### 5) Camel Quarkus
+
+The Camel Quarkus project is working on a new release with Camel 4 support, expected later this month.
+Stay tuned for their release announcements.
+
+### 6) Camel Main
+
+For users that just want to run standalone Camel, then we have `camel-main` that are constantly being improved.
```

Comment:
How about:
For users who just want to run Camel as a standalone application, we have `camel-main` which is constantly being improved.